src/CryptoAxus.Application/Dto/BaseDto.cs

```diff
@@ -4,13 +4,26 @@ public abstract class BaseDto
 {
     public ObjectId? Id { get; set; }
 
-    public DateTime? CreatedDate { get; set; }
+    public DateTime? CreatedDate { get; set; } = DateTime.UtcNow;
```

Comment:
This value should not be set in the `BaseDto`.

Reply:
Resolved